build/package/config.yaml

```diff
@@ -0,0 +1,35 @@
+pipeline:
+  # An example input that generates a single log entry when Stanza starts up.
+  - type: generate_input
+    count: 1
+    entry:
+      record: This is a sample log generated by Stanza
+    output: example_output
+
+  # An example input that monitors the contents of a file.
+  # For more info: https://github.com/observIQ/stanza/blob/master/docs/operators/file_input.md
+  #
+  # - type: file_input
+  #   include:
+  #     - /sample/file/path
+  #   output: example_output
+
+  # An example output that sends captured logs to stdout.
+  - id: example_output
+    type: stdout
+
+  # An example output that sends captured logs to google cloud logging.
+  # For more info: https://github.com/observIQ/stanza/blob/master/docs/operators/google_cloud_output.md
+  #
+  # - id: example_output
+  #   type: google_cloud_output
+  #   credentials_file: /my/credentials/file
+
+  # An example output that sends captured logs to elasticsearch.
+  # For more info: https://github.com/observIQ/stanza/blob/master/docs/operators/elastic_output.md
+  #
+  # - id: example_output
+  #   type: elastic_output
+  #   addresses:
+  #     - http://my_node_address:9200
+  #   api_key: my_api_key
```

Comment:
newline

Reply:
Everytime. I swear I went through and checked :upside_down_face: